src/Microsoft.DotNet.XHarness.CLI/Commands/Android/AndroidRunCommand.cs

```diff
@@ -194,7 +194,7 @@ protected override Task<ExitCode> InvokeInternal(ILogger logger)
 
                     if (processCrashed)
                     {
-                        runner.DumpBugReport(Path.Combine(outputDirectory, $"adb-bugreport-{apkPackageName}.zip"));
+                        runner.DumpBugReport(Path.Combine(outputDirectory, $"adb-bugreport-{apkPackageName}"));
```

Comment:
I think it might be a bit surprising for the user that the file won't be available on the path he provided in `outputFilePath`.

If we append a suffix, I would return the file name from the method so it's clear. Maybe even think about the name of the argument to be more explicit.

Reply:
Why won't it be provided? I put the file there in both cases, the only difference is format. The name and format wasn't provided by user as well, we always generated it by ourselves and then put notify in the output.

Reply:
I mean it with regards to this specific API/method - you call the method:
```
runner.DumpBugReport("output/adb-bugreport-net.dot.whatever");
```

And after you call it, you would expect a file on a path that you supplied to an argument called `outputFilePath`.
But there won't be a file like that, there will be some `adb-bugreport-net.dot.whatever.zip` or `.txt` and if you want to copy it, read it.. it will be very complicated.

I understand we just put it in some directory that gets uploaded automatically and we don't care for the specific name in our case but that might be beyond the scope of the API of the method.

So my only suggestion is to change the method to return `string` instead of `void` and give us back the name of the file it actually produced. Possibly also change the name of the parameter a little so it's clear it's just a hint where to create the file.
This way, you will have a more intuitive API of the method as when you want to use it, you will know there's some string coming back and you will look into what it is.